pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.poetry]
+name = "superset-ext"
+version = "0.0.1"
+description = "`superset-ext` is a Meltano utility extension."
+authors = ["Meltano Team and Contributors"]
+maintainers = ["Meltano Team and Contributors"]
+license = "Apache 2.0"
+
+[tool.poetry.dependencies]
+python = "<3.11,>=3.7"
+structlog = "^21.2.0"
+PyYAML = "^6.0.0"
+pydantic = "^1.9.0"
+click = "^8.1.3"
+typer = "^0.6.1"
+"meltano.edk" = {git = "https://github.com/meltano/edk.git", rev="main"}
+
+[tool.poetry.dev-dependencies]
+black = "^22.3.0"
+isort = "^5.10.1"
+flake8 = "^3.9.0"
+
+[build-system]
+requires = ["poetry-core>=1.0.8"]
+build-backend = "poetry.core.masonry.api"
+
+packages = [
+    { include = "superset_ext" }
+    # uncomment this if you want to include static assets
+    # { include = "files_superset-ext.*" }
+]
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3 # Vertical Hanging Indent
+src_paths = "superset_ext"
+
+# If you need to static assets with your extension, you can add them here.
+# Under a directory path of "files_superset-ext/"
+#include = [
+#    "files_superset-ext/somefile.thing"
+#]
```

Comment:
no static assets for superset required 🥳

Reply:
@pandemicsyn we debated about something like secret key generation during the original implementation but I think the sentiment was that we'd prefer to tell users to run a few explicit commands vs being overly smart under the hood.

My personal view is that in some cases we should otherwise were simply wrapping the existing commands which doesnt end up adding a lot of value. But it seems like thats a debate for a future iteration. I think sticking with the status quo for this iteration is fine.